code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -24,8 +24,9 @@
 	var/sport = 1 //High sport helps you show off on a treadmill. Multiplicative
 	var/custom_metabolism = REAGENTS_METABOLISM
 	var/custom_plant_metabolism = HYDRO_SPEED_MULTIPLIER
-	var/overdose = 0
-	var/overdose_dam = 1
+	var/overdose_am = 0
+	var/overdose_tick = 0
+	var/tick
```

Comment:
Data's a list and used for other things such as blood transfusion. there's nothing that just naturally ticks upwards unless specified (like in your example, where data is reset to being an integer)